### PR TITLE
adding Quake binary and usage information

### PR DESCRIPTION
**Pull request by Korin777**
use make quake，then it will download quake shareware and run quake

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -77,16 +77,27 @@ $(OUT)/DOOM1.WAD:
 	sha1sum -c $@.sha1
 	$(RM) shareware_doom_iwad.zip
 
+Quake_shareware = https://www.libsdl.org/projects/quake/data/quakesw-1.0.6.zip
+$(OUT)/id1/pak0.pak:
+	$(VECHO) " Downloading $@ ...\n"
+	wget $(Quake_shareware)
+	unzip -d $(OUT) quakesw-1.0.6.zip
```
Comment (jserv) [inline Makefile:84]: Do check the critical file(s) via `sha1sum`.

**Comment by jserv** [comment]
For the sake of performance, you should build Quake with `-O3` optimization. In addition, mention the repository of modified Quake source code.

File: Makefile [hunk 2]
```diff
@@ -77,16 +77,27 @@ $(OUT)/DOOM1.WAD:
...
 	(cd $(OUT); ../$(BIN) doom.elf)
 
+quake: $(BIN) $(OUT)/id1/pak0.pak
+	(cd $(OUT); ../$(BIN) quake.elf)
+
 clean:
 	$(RM) $(BIN) $(OBJS) $(deps)
 distclean: clean
 	$(RM) $(OUT)/DOOM1.WAD $(OUT)/DOOM1.WAD.sha1
+	rm -r $(OUT)/id1
```
Comment (jserv) [inline Makefile]: use `$(RM)` since it allows overriding by users.

**Comment by Korin777** [comment]
Is that mention the repository of modified Quake here?

repository of modified Quake source code : https://github.com/Korin777/quakembd

**Comment by Korin777** [comment]
> For the sake of performance, you should build Quake with `-O3` optimization. In addition, mention the repository of modified Quake source code.

I have already built quake with -O3!

Is that mention the repository of modified Quake source code here?

repository of modified Quake source code : https://github.com/Korin777/quakembd

**Comment by jserv** [comment]
Thank @Korin777 for contributing!

**Comment by Korin777** [comment]
Thanks professor for giving me this opportunity!

**Pull request merged**